.github/workflows/solidity-tests.yml

```diff
@@ -0,0 +1,45 @@
+name: Solidity Tests CI
+
+on:
+  pull_request:
+    branches: [main, release/**]
+  push:
+    branches: [main, release/**]
+    tags: [v*]
+
+concurrency:
+  group: tests-${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: write
+  checks: write
+  pull-requests: write
+
+jobs:
+  BlockInfo:
```

Comment:
```suggestion
  SolidityEquivalence:
```

Reply:
By doing this I assume we are going to run all of the solidity equivalence tests in one job.  Is that what we want?  I am not sure that's scalable.

Reply:
You can break them out but it shouldn't be by file.
We can group them in larger buckets.
What would you suggest?

Reply:
I guess the term "SolidityEquivalence" is what I am questioning here.  I think it makes sense to start with this, so I'll make the update, but the term seems to be an umbrella term for this whole effort which will result in many more tests.

Reply:
Done.